lib-multisrc/colamanga/src/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt

```diff
@@ -47,25 +63,31 @@ abstract class ColaManga(
 
     override val supportsLatest = true
 
-    private val json: Json by injectLazy()
-
-    private val intl = ColaMangaIntl(lang)
-
+    private var pages: MutableMap<String, ArrayList<Page>> = mutableMapOf()
+    private val intl = Intl(
+        Locale.getDefault().language,
+        setOf("en", "zh"),
+        lang,
+        javaClass.classLoader!!,
+    )
     private val preferences by getPreferencesLazy()
-
-    override val client = network.cloudflareClient.newBuilder()
-        .rateLimitHost(
-            baseUrl.toHttpUrl(),
-            preferences.getString(RATE_LIMIT_PREF_KEY, RATE_LIMIT_PREF_DEFAULT)!!.toInt(),
-            preferences.getString(RATE_LIMIT_PERIOD_PREF_KEY, RATE_LIMIT_PERIOD_PREF_DEFAULT)!!.toLong(),
-            TimeUnit.MILLISECONDS,
-        )
-        .addInterceptor(ColaMangaImageInterceptor())
-        .build()
-
-    override fun headersBuilder() = super.headersBuilder()
-        .add("Origin", baseUrl)
-        .add("Referer", "$baseUrl/")
+    var webView: MutableMap<String, WebView> = mutableMapOf()
+    val interfaceName = randomString()
```

Comment:
Use a more specific name like `webViewCache` or `webViewMap`.

I think these two fields can be private.